runtime/oti/vm_api.h

```diff
@@ -2343,6 +2343,56 @@ UDATA
 findIndexInFlattenedClassCache(J9FlattenedClassCache *flattenedClassCache, J9ROMNameAndSignature *nameAndSignature);
 #endif /* J9VM_OPT_VALHALLA_VALUE_TYPES */
 
+/**
+ * Returns the offset of a qtype field.
+ * 
+ * @param[in] fieldOwner the J9class that defines the field
+ * @param[in] field romfieldshape of the field
+ * @param[in] instance flag indicate if its an instance or static field
+ *
+ * @return field offset
+ */
+UDATA
+getFlattenableFieldOffset(J9Class *fieldOwner, J9ROMFieldShape *field, BOOLEAN instanceField);
```

Comment:
We dont need the BOOLEAN instanceField param, we should be able to tell since all field names will be unique. See https://github.com/eclipse/openj9/issues/9880. Currently, only instance fields are qtypes, so dont worry about statics for now.

Reply:
this goes for the rest of the helpers